tests/utils.sh

```diff
@@ -2,8 +2,8 @@
 
 get_templates() {
     echo "Getting latest confd templates from calicoctl repo"
-    git clone https://github.com/projectcalico/calicoctl.git /calicoctl
-    ln -s /calicoctl/calico_node/filesystem/etc/calico/ /etc/calico
+    git clone https://github.com/projectcalico/calico.git /node-repo
```

Comment:
These aren't cloned into the repo, just into the docker container running the tests